### PR TITLE
Updated CHANGELOG and pubspec.yalm

### PR DESCRIPTION
**Pull request by Mmisiek**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: flutter_quill
 description: A rich text editor supporting mobile and web (Demo App @ bulletjournal.us)
-version: 7.1.17+1
+version: 7.1.18+1
```
Comment (adil192) [inline pubspec.yaml:3]: This should be `7.1.18` or `7.2.0` depending on what @singerdmx prefers

**Comment by adil192** [comment]
Also I think you need to branch off the latest singerdmx/flutter-quill commit rather than just using your previous branch from #1210

**Comment by Mmisiek** [comment]
I agree but I honestly don't have time to deal with it now. Please feel free to do it correctly.

**Comment by adil192** [comment]
Nevermind looks like this has already been done in https://github.com/singerdmx/flutter-quill/commit/0a3fe3f63c4636c68de1e44523eae3b9de787dcb

This PR can be closed

**Pull request closed without merging**